css-conditional-5/Overview.bs

```diff
@@ -1206,11 +1227,15 @@ Sticky positioning: the '@container/overflowing' feature</h4>
 
 	The '@container/overflowing' [=container feature=] queries whether a
 	[=scroll container=] has [=scrollable overflow=] in the given direction for the
-	current scroll progress. The logical values map to physical based on the direction
-	and writing-mode of the [=query container=].
+	current scroll progress. Content not reachable through user scrolling, like
```

Comment:
Hmmm, now that I'm rereading this text and your replies... what does "current scroll progress" mean here.  Does it mean that an LTR overflow:scroll container is `overflowing(left)` once it's been scrolled away from its initial horizontal position?  (That contradicts the following sentence, which I had suggested you add!)

Reply:
Reworded.